requirements.in

```diff
@@ -3,6 +3,7 @@ authlib==1.2.0
 black==23.1.0
 cattrs==22.2.0
 click==8.1.3
+cryptography==39.0.1
```

Comment:
hm, I don't love pinning this when it was previously installed thru `authlib`, but I can't think of a good alternative if `authlib` isn't updating the version

Reply:
I did regenerate it yesterday in this PR and it was still tagging the old (our current version):
https://github.com/mozilla/bigquery-etl/pull/3577/files

Reply:
Just tried recompiling the requirements again without explicitly tagging cryptography and it does not update. If you prefer we can also wait until a new version of `authlib` is released @sean-rose

Reply:
I think `pip-compile` tries to generally preserve what's in `requirements.txt` unless it feels it has a compelling reason to update.  If you remove the `cryptography` entry from `requirements.txt` and re-compile it will pick `cryptography` 39.0.1 (without needing the pin in `requirements.in`).

Reply:
ok, tried recompiling today and it appears we got the desired effect.